rosdep/base.yaml

```diff
@@ -602,6 +602,17 @@ dnsmasq:
   fedora: [dnsmasq]
   gentoo: [net-dns/dnsmasq]
   ubuntu: [dnsmasq]
+docker:
+  debian: [docker]
+  fedora: [docker]
+  ubuntu: [docker]
+docker-compose:
+  debian: [docker-compose]
+  fedora: [docker-compose]
+  ubuntu: [docker-compose]
+docker.io:
+  debian: [docker.io]
```

Comment:
From the link it appears that docker.io is only available on buster and newer. We should explicitly set None for the other currently supported distros so people don't get surprised when it doesn't work. And use a wildcard for buster and newer.

Reply:
Is this correct now?